docs/man/docker-cp.1.md

```diff
@@ -1,17 +1,50 @@
 % DOCKER(1) Docker User Manuals
 % Docker Community
-% JUNE 2014
+% JANUARY 2015
 # NAME
-docker-cp - Copy files/folders from the PATH to the HOSTPATH
+docker-cp - Copy files/folders between containers or a local path
 
 # SYNOPSIS
 **docker cp**
-CONTAINER:PATH HOSTPATH
+[SRC_CONTAINER:]SRC_PATH [DST_CONTAINER:]DST_PATH
 
 # DESCRIPTION
-Copy files/folders from a container's filesystem to the host
-path. Paths are relative to the root of the filesystem. Files
-can be copied from a running or stopped container.
+Copy files/folders to or from a container and your local host
+or between any two containers. Container paths are relative to
+the container's working directory. Files can be copied to and
```

Comment:
re: relative to the container's working dir - is there a way for a client to query for the current working dir?  I guess they could do 'docker exec ... pwd' but that feels hacky.  I'm asking because it seems like the notion of "working dir" is really only brought up in the context of Dockerfile processing.  If we start expecting people to use it outside of the build process then I think we may need to ask whether or not there should be first class ways to query and modify it.
I guess I'm wondering if we should require the path to be an absolute path, at least for now?